lib/components/Input/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./base";
```

Comment:
Shouldn't the directory be called 'input' instead of 'Input'?
*@jia-deriv gave us an example with dir's and files named in kebab-case.

Reply:
nope. Component directory can be `input`. we are following deriv-ui

Reply:
Sorry, my bad we should use pascal case instead of kebab-case if we are following deriv-ui